deployment/deploy/2120-deploy-FallbackOracle.js

```diff
@@ -0,0 +1,19 @@
+const path = require("path");
+const col = require("cli-color");
+//const deploymentName = getContractNameFromScriptFileName(path.basename(__filename));
+const func = async function (hre) {
+    const {
+        deployments: { deploy, log },
+        getNamedAccounts,
+    } = hre;
+    const { deployer } = await getNamedAccounts(); //await ethers.getSigners();
+    log(col.bgYellow("Deploying FallbackOracle..."));
+    await deploy("FallbackOracle", {
+        from: deployer,
+        args: [],
+        log: true,
+        skipIfAlreadyDeployed: true,
+    });
```

Comment:
rename the deployment script file to `2120-deploy-DummyFallbackOracle.js`
and
```suggestion
    await deploy("FallbackOracle", {
        contract: "DummyFallbackOracle",
        from: deployer,
        args: [],
        log: true,
        skipIfAlreadyDeployed: true,
    });
```

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/549/commits/d47e661d64be04ec9d03dbffbe6d7b0dd835ca50)